Client/Extensions/UIView+Extension.swift

```diff
@@ -56,15 +65,16 @@ extension UIView {
     func addBlurEffect(using style: UIBlurEffect.Style) {
         let blurEffect = UIBlurEffect(style: style)
```

Comment:
Should we check `isReduceTransparencyEnabled` here as well?

Reply:
yes good point cause is public and can be used independently